src/Php80/Resources/stubs/PhpToken.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+if (PHP_VERSION_ID < 80000 && !class_exists('PhpToken') && function_exists('token_get_all')) {
```

Comment:
no need for the class_exists check, and better use extension_loaded which can be optimized by opcache:
`if (PHP_VERSION_ID < 80000 && extension_loaded('tokenizer')) {`